api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -711,6 +718,30 @@ public void testCreateWorkspace_accessTierIgnored_grantWorkflowPermission_contro
     verify(mockIamService).grantWorkflowRunnerRoleToCurrentUser(DEFAULT_GOOGLE_PROJECT);
   }
 
+  @Test
+  public void testCreateWorkspaceAsync() {
+    Workspace workspace = createWorkspace();
+    WorkspaceOperation operation = workspacesController.createWorkspaceAsync(workspace).getBody();
+    assertThat(operation.getId()).isNotNull();
+    assertThat(operation.getStatus()).isEqualTo(WorkspaceOperationStatus.PENDING);
+    assertThat(operation.getWorkspace()).isNull();
+  }
+
+  // TODO!
```

Comment:
it's going to take a bit of effort getting the dependencies right for this test ... I'll try to get to it soon.

Reply:
nit: probably would leave jira ticket in the TODO here as well